config/envs/default.py

```diff
@@ -103,6 +103,7 @@ class DefaultConfig:
     ASSESSMENT_SCORES_ENDPOINT = ASSESSMENT_STORE_API_HOST + "/score"
 
     ASSESSMENT_COMMENT_ENDPOINT = ASSESSMENT_STORE_API_HOST + "/comment"
+    ASSESSMENT_PROGRESS = ASSESSMENT_STORE_API_HOST + "/bulk-progress"
```

Comment:
Can we call this `ASSESSMENT_PROGRESS_ENDPOINT` pls?

Reply:
Will do 👍

Reply:
Done!